image-magick.yaml

```diff
@@ -0,0 +1,38 @@
+package:
+  name: image-magick
+  version: 18
+  epoch: 0
+  description: "library for displaying, converting, and editing raster image and vector image files"
+  copyright:
+    - license: ImageMagick Studio LLC
```

Comment:
I think we should use the SPDX identifier, which is just "ImageMagick" in this case.

Reply:
They mention that`the license is compatible with the GPL V3` in https://imagemagick.org/script/license.php we can use GPLV3

Reply:
I changed the license as GPLV3